docs/guides/distribution/updater.md

```diff
@@ -2,220 +2,139 @@
 sidebar_position: 5
 ---
 
-# Updater
-
-## Configuration
-
-Once you have your Tauri project ready, you need to configure the updater.
-
-Add this in tauri.conf.json
-
-```json
-"updater": {
-    "active": true,
-    "endpoints": [
-        "https://releases.myapp.com/{{target}}/{{current_version}}"
-    ],
-    "dialog": true,
-    "pubkey": "YOUR_UPDATER_SIGNATURE_PUBKEY_HERE"
-}
-```
-
-The required keys are "active", "endpoints" and "pubkey"; others are optional.
-
-"active" must be a boolean. By default, it's set to false.
-
-"endpoints" must be an array. The string `{{target}}` and `{{current_version}}` are automatically replaced in the URL allowing you to determine [server-side](#update-server-json-format) if an update is available. If multiple endpoints are specified, the updater will fallback if a server is not responding within the pre-defined timeout.
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+import Command from '@theme/Command'
+import updaterDialogImage from '/img/guides/distribution/updater/update-available.png'
 
-"dialog" if present must be a boolean. By default, it's set to true. If enabled, [events](#events) are turned off as the updater handles everything. If you need the custom events, you MUST turn off the built-in dialog.
-
-"pubkey" must be a valid public-key generated with Tauri CLI. See [Signing updates](#signing-updates).
-
-### Update Requests
+# Updater
 
-Tauri is indifferent to the request the client application provides for update checking.
+Tauri offers a built-in updater for the MSI (Windows), AppImage (Linux) and App bundle (macOS) distribution formats.
 
-`Accept: application/json` is added to the request headers because Tauri is responsible for parsing the response.
+Once your Tauri project is ready, you can configure Tauri's updater to enable auto updating for your users.
 
-For the requirements imposed on the responses and the body format of an update, response see [Server Support](#server-support).
+## Signing updates
```

Comment:
```suggestion
## Signing Updates
```

Even added it to the contributing section 😂

Reply:
ffs

Reply:
eeehhhh, i mean, i obviously just wanted to check how thorough you review my PRs 😂

Reply:
Haha you caught me on a good day 😂